core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -39,6 +39,10 @@ import scala.jdk.CollectionConverters._
 import scala.collection.Seq
 import scala.collection.mutable.Set
 
+object ZooKeeperClient {
+  val AuthFailedRetryBackoffMs = 100
```

Comment:
In reinitialize(), we back off by 1 sec if the creation of ZooKeeper instance fails. Perhaps we should make this 1 sec too to be more consistent?

Reply:
Done.